presto-hive/src/test/java/com/facebook/presto/hive/TestShowStats.java

```diff
@@ -32,6 +32,7 @@ public TestShowStats()
     public void setUp()
     {
         assertUpdate("CREATE TABLE nation_partitioned(nationkey BIGINT, name VARCHAR, comment VARCHAR, regionkey BIGINT) WITH (partitioned_by = ARRAY['regionkey'])");
+        assertUpdate("CREATE TABLE orders(orderkey BIGINT, totalprice BIGINT, categoryid BIGINT) WITH (partitioned_by = ARRAY['categoryid'])");
```

Comment:
I understand that `orders` needs to be a valid table. However, this table is partitioned, but the name doesn't have `_partitioned` suffix. Also, orders table comes from TPC-H schema where it has different set of columns. To avoid confusion, let's create `orders` and `lineitem` tables to match TPC-H: `CREATE TABLE orders (LIKE tpch.tiny.orders)` and `CREATE TABLE lineitem (LIKE tpch.tiny.lineitem)`. Also, let's put these statements after `INSERT INTO nation_partitioned` to avoid splitting logic related to `nation_partitioned` table.

Reply:
Tried "CREATE TABLE orders_partitioned(LIKE tpch.tiny.nation) WITH (partitioned_by = ARRAY['orderstatus'])" ,    but got error: java.lang.RuntimeException: line 1:1: LIKE table across catalogs is not supported